.github/workflows/check.yaml

```diff
@@ -39,4 +39,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          args: --issues-exit-code=0 -E gofmt
+          args: -E gofmt -D errcheck,unused,structcheck
```

Comment:
I would add a short recipe how to run it locally. Say, a link to the [installation instruction](https://golangci-lint.run/usage/install/) and a command equivalent to the check in CI (for me it is `~/go/bin/golangci-lint run -E gofmt -D errcheck,unused,structcheck`).